src/main/java/club/sk1er/hytilities/handlers/chat/modules/events/APIKeyEvent.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Hytilities - Hypixel focused Quality of Life mod.
+ * Copyright (C) 2021  Sk1er LLC
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package club.sk1er.hytilities.handlers.chat.modules.events;
+
+import club.sk1er.hytilities.Hytilities;
+import club.sk1er.hytilities.config.HytilitiesConfig;
+import club.sk1er.hytilities.handlers.chat.ChatReceiveModule;
+import club.sk1er.hytilities.util.APIUtils;
+import club.sk1er.mods.core.util.Multithreading;
+import net.minecraft.util.EnumChatFormatting;
+import net.minecraftforge.client.event.ClientChatReceivedEvent;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class APIKeyEvent implements ChatReceiveModule {
+
+    @Override
+    public void onMessageReceived(@NotNull ClientChatReceivedEvent event) {
+        /*/
+            Adapted from Moulberry's NotEnoughUpdates, under the Attribution-NonCommercial 3.0 license.
+            https://github.com/Moulberry/NotEnoughUpdates
+         */
```

Comment:
Nitpicking, but you didn't follow convention at all here, even though you did [here](https://github.com/Sk1erLLC/Hytilities/pull/83/files#diff-ab29602b2a15ed3653bb310fd8db6f4293877abc7c38d272929f5a714852a91aR34).